### PR TITLE
msg consume latency stats

### PR DESCRIPTION
**Pull request by DoraALin**
channel consume latency distribution in channel page.
Signed-off-by: 元守 <lulin@youzan.com>

### CODE REVIEW

File: nsqd/stats.go [hunk 1]
```diff
@@ -285,12 +287,43 @@ type TopicMsgStatsInfo struct {
...
+	latencyStats := make([]int64, len(self.MsgConsumeLatencyStats))
+	for i, _ := range self.MsgConsumeLatencyStats {
+		latencyStats[i] = atomic.LoadInt64(&self.MsgConsumeLatencyStats[i])
+	}
+	return latencyStats
+}
+
+const NANO_TO_MS = 1024 * 1024;
+
+func (self *ChannelStatsInfo) UpdateChannelLatencyStats(latency int64) {
```
Comment (absolute8511) [inline nsqd/stats.go]: Maybe treat the latency as `microsecond`  just the same as topic stats

File: nsqd/channel.go [hunk 2]
```diff
@@ -843,6 +849,7 @@ func (c *Channel) internalFinishMessage(clientID int64, clientAddr string,
 	if c.e2eProcessingLatencyStream != nil {
 		c.e2eProcessingLatencyStream.Insert(msg.Timestamp)
 	}
+	c.channelStatsInfo.UpdateChannelStats(time.Now().UnixNano() - msg.Timestamp)
```
Comment (absolute8511) [inline nsqd/channel.go]: Maybe use `microsecond` just like the topic latency stats

**Comment by absolute8511** [comment]
#50

**Comment by DoraALin** [comment]
@absolute8511 for your review

**Review by absolute8511 — APPROVED** [review]
LGTM

**Pull request merged**